### PR TITLE
[WIP] 8.0 stock cycle count backport

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by daramousk** [comment]
@lreficent This depends on another module that has been provided on another PR. Is there something to be done for the runbot to get past this?

**Comment by LoisRForgeFlow** [comment]
@daramousk Yes, you can add temporarily other PRs on the oca_dependencies.txt file. See an example here: https://github.com/OCA/manufacture/pull/199/commits/281ad10793c4d986e0ca15b9e4d40e9dca9fd6e9

**Comment by daramousk** [comment]
@lreficent All greened

**Comment by LoisRForgeFlow** [comment]
Adding the WIP label as there is still dependencies with open PRs.

For the record:

* [x] https://github.com/OCA/stock-logistics-warehouse/pull/420
* [x] https://github.com/OCA/stock-logistics-warehouse/pull/422

**Comment by daramousk** [comment]
I think #420 should be fine now.

**Comment by daramousk** [comment]
@lreficent #420 has been merged

**Review by rousseldenis — APPROVED** [review]
Code review

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 stock-logistics-tracking
 stock-logistics-barcode
 stock-logistics-workflow
+stock-logistics-warehouse https://github.com/daramousk/stock-logistics-warehouse.git 8.0-stock_inventory_discrepancy-port
+stock-logistics-warehouse-01 https://github.com/daramousk/stock-logistics-warehouse.git 8.0-mig-stock_inventory_exclude_sublocation
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: you can now remove these 2 lines

**Comment by daramousk** [comment]
@lreficent Done. Not sure about the travis error though.

**Comment by LoisRForgeFlow** [comment]
@daramousk I just restarted the build and it went right this time!

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks, LGTM :+1:

**Pull request merged**